### PR TITLE
Update systemd files to not mention config.json file

### PR DESCRIPTION
**Pull request by nitisht**

### CODE REVIEW

File: linux-systemd/distributed/minio.service [hunk 1]
```diff
@@ -15,6 +15,8 @@ PermissionsStartOnly=true
 
 EnvironmentFile=-/etc/default/minio
 ExecStartPre=/bin/bash -c "[ -n \"${MINIO_VOLUMES}\" ] || echo \"Variable MINIO_VOLUMES not set in /etc/defaults/minio\""
+ExecStartPre=/bin/bash -c "[ -n \"${MINIO_ACCESS_KEY}\" ] || echo \"Variable MINIO_ACCESS_KEY not set in /etc/defaults/minio\""
+ExecStartPre=/bin/bash -c "[ -n \"${MINIO_SECRET_KEY}\" ] || echo \"Variable MINIO_SECRET_KEY not set in /etc/defaults/minio\""
```
Comment (harshavardhana) [inline linux-systemd/distributed/minio.service]: Only one entry is needed.

Comment (nitisht) [reply linux-systemd/distributed/minio.service]: you mean chain the conditions in one entry?

Comment (harshavardhana) [reply linux-systemd/distributed/minio.service]: No why are you writing ExecStartPre ?

Comment (harshavardhana) [reply linux-systemd/distributed/minio.service]: We should write it in one line..

**Comment by nitisht** [comment]
Tested locally and updated the PR, PTAL

**Pull request merged**